src/Map.js

```diff
@@ -2,7 +2,7 @@ import L from 'leaflet';
 import tileLayer from './layers/TileLayer';
 import wmsLayer from './layers/WmsLayer';
 import googleLayer from './layers/GoogleLayer';
-import geoJson from './layers/GeoJson';
+import geoJson from './layers/GeoJsonGroup';
```

Comment:
Should this variable be called geoJsonGroup?

Reply:
Yes, it could, but then I would also need to change it for layerTypes which will influence the Map app.